plaidml/bridge/openvino/src/plaidml_plugin/ops/relu.cpp

```diff
@@ -4,23 +4,21 @@
 
 #include "plaidml_ops.hpp"
 
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
 #include "plaidml/op/op.h"
 
 using namespace plaidml;          // NOLINT[build/namespaces]
 using namespace InferenceEngine;  // NOLINT[build/namespaces]
 
 namespace PlaidMLPlugin {
 
-IE_SUPPRESS_DEPRECATED_START
-
 static OpRegistration reg("relu", [](const Context& ctx) {
-  auto* layer = dynamic_cast<ReLULayer*>(ctx.layer);
+  // Note that nGraph ReLU does not include an alpha parameter
```

Comment:
hopefully this would be dealt with somewhere in Model Optimizer by adding a multiplication, doesn't mean we can safely ignore but would be good to confirm

Reply:
nGraph has a PReLU layer for cases with an alpha parameter, presumably that will be called instead.